src/System.Collections/tests/Generic/SortedSet/SortedSet.Generic.Tests.cs

```diff
@@ -60,22 +60,37 @@ public void SortedSet_Generic_Constructor_IEnumerable_Null()
         public void SortedSet_Generic_Constructor_IEnumerable_IComparer(EnumerableType enumerableType, int setLength, int enumerableLength, int numberOfMatchingElements, int numberOfDuplicateElements)
         {
             IEnumerable<T> enumerable = CreateEnumerable(enumerableType, null, enumerableLength, 0, 0);
+#if netcoreapp
```

Comment:
Actually in this particular case I'm not so sure it's worth it separating to a different file given that it is only a couple of lines and it is a behavioral difference. I know that what we try is to avoid ifdefs when possible, but I'm not so sure that it is worth it in cases like this. That said I don't have a strong opinion, so if you still think we should separate it I'm fine.